include/openssl/des.h

```diff
@@ -77,6 +77,8 @@ typedef struct DES_cblock_st {
   uint8_t bytes[8];
 } DES_cblock;
 
+typedef struct DES_cblock_st const_DES_cblock;
```

Comment:
Why is it called `const_DES_cblock` if it isn't defined using `const` in the typedef?

Reply:
I'm not sure why.  The equivalent definitions provided by [OpenSSL](https://github.com/openssl/openssl/blob/18f2091ad14c33721fc6de3e0f9b1cdfe5bffcf6/include/openssl/des.h#L36) and [Libgit2](https://github.com/libgit2/libgit2/blob/eb22b600633ba172791f5d4e35ee8e01a4c3525d/deps/ntlmclient/crypt_openssl.h#L24) are likewise not `const`.